Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -443,6 +448,24 @@ def fetch_resource(formula, new_version, url, **specs)
     [resource.fetch, forced_version]
   end
 
+  def update_pypi_url(old_url, version)
+    formula_name = File.basename(old_url).match(/^(.+)-[a-z0-9.]+$/)[1]
+    metadata_url = "https://pypi.org/pypi/#{formula_name}/json"
```

Comment:
We probably should use `http://pypi.org/pypi/<package_name>/<version>/json` instead

Source: https://wiki.python.org/moin/PyPIJSON

Reply:
`http://pypi.org/pypi/<package_name>/json` and `http://pypi.org/pypi/<package_name>/<version>/json` seem to return the same file

Tested locally with `http://pypi.org/pypi/roundup/json` and `https://pypi.org/pypi/roundup/2.0.0/json`

Reply:
Ah, it returns the same because I'm fetching the most recent version.

Do you think this is necessary? I think the line below should take care of that:

```ruby
json["releases"][version].find { |url| url["packagetype"] == "sdist" }["url"]
```

Reply:
I haven’t checked, but I think the current URL returns a JSON of all versions, so using the version-specific URL might save bandwidth

Reply:
Take a look at https://warehouse.readthedocs.io/api-reference/json/. It looks like it changes which version's info is shown at the top level but still contains all of the releases